.github/workflows/tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        php-version:
+          - 7.2
+          - 7.3
+          - 7.4
+        symfony-version:
+          - 4.4.0
+          - ^5.0
+        experimental: [false]
+        include:
+          - php-version: 7.1
+            symfony-version: 4.4.0
+            experimental: false
+          - php-version: 8.0
+            symfony-version: ^5.0
+            experimental: true
```

Comment:
I suggest to remove this section as this package is currently not compatible with PHP 8.x branch.

```suggestion
```